### DISCUSSION TITLE
Supps

### DISCUSSION THREAD

**Discussion by janply** [NAIST-SE/AttractingContributors#627] [General]
Anyone takes supplements all the time? Are they necessary?

**Comment by McJul** [comment]
I do take vitamins regularly. However, it's not random choice. I first have my blood test done and only after that I follow my doc's suggestions about the choice of supplements. I currently take Vitamin D3, Omega 3, Magnesium and some [Canadian pharmacy](https://www.canadapharmacy.com/) supplements my doc prescribed me. You shouldn't take any supplements without consulting a professional. Each body is unique and you should know what it needs exactly.
